tests/providers/samba/hooks/test_samba.py

```diff
@@ -35,6 +35,12 @@
     password="password",
 )
 
+NULL_HOST_SCHEMA_CONNECTION = Connection(
+    host=None,
+    schema=None,
+    login="username",
+    password="password",
+)
```

Comment:
```suggestion
)

```

To fix static checks